### DISCUSSION TITLE
Functionality similar to VSCodes undo. Revert alternative.

### DISCUSSION THREAD

**Discussion by ttytm** [jesseduffield/lazygit#3106] [Q&A] [answered]
I'm looking for a functionality similar to VSCodes `Git: Undo Last Commit`.

Reverting would just add another commit that reverted the changes. But this one removes the last commit from the log and stages the changes that were part of the commit that was undone.

**Comment by mark2185** [comment, marked as answer]
Try going to the commits panel, selecting the commit you wish to `git reset` to and press `g`. You want a `soft reset`, I presume.

**Reply by ttytm** [reply]
Thanks @mark2185 :+1:
